addons/logistics_wirecutter/XEH_preInit.sqf

```diff
@@ -10,3 +10,29 @@ PREP(interactEH);
 PREP(isFence);
 
 ADDON = true;
+
+
+// Condition for Interaction
+DFUNC(Condition) = {
```

Comment:
i think its not good to move condition and statement in the original file because both are defined in a PFH and this on each Frame